plugin/dnstap/setup.go

```diff
@@ -38,6 +39,26 @@ func parseConfig(c *caddy.Controller) (Dnstap, error) {
 	}
 
 	d.IncludeRawMessage = c.NextArg() && c.Val() == "full"
+	
+	d.Identity, _ = os.Hostname()
+	d.Version = caddy.AppName + "-" + caddy.AppVersion 
+	
+	for c.NextBlock() {
+		switch c.Val() {
+			case "identity": {
+				if !c.NextArg() {
+					return d, c.ArgErr()
+				}
+				d.Identity = c.Val()
+			}
+  		case "version": {
+				if !c.NextArg() {
+					return d, c.ArgErr()
+				}
+				d.Version = c.Val()
+			}
+		}
+	}
```

Comment:
The formatting here looks a bit odd, did you use `go fmt`?

Reply:
`go fmt` did not complain about this, but I agree! The indentation on github looks different to what I had in my editor.